src/main/kotlin/com/shortener/service/UrlServiceImpl.kt

```diff
@@ -24,7 +25,8 @@ private val baseUrl = ServletUriComponentsBuilder.fromCurrentContextPath().build
 class UrlServiceImpl(
     private val urlEntryRepository: UrlEntryRepository,
```

Comment:
why do we use both `UrlEntryRepository` and `UrlEntryRepositoryBase`? 

`UrlEntryRepository` is a child of `UrlEntryRepositoryBase`, so this looks awkward

Reply:
Further to the answer from the question above: `UrlEntryRepositoryBase` is defining the methods which are cached(well, I need to rename it)